bitcoin/src/psbt/raw.rs

```diff
@@ -124,10 +124,17 @@ impl Serialize for Pair {
     }
 }
 
-impl Decodable for Pair {
-    fn consensus_decode<R: io::Read + ?Sized>(r: &mut R) -> Result<Self, encode::Error> {
+impl Deserialize for Pair {
+    fn deserialize(bytes: &[u8]) -> Result<Self, encode::Error> {
+        let mut decoder = Cursor::new(bytes);
```

Comment:
Again, shouldn't be needed.

Reply:
I plan to address all of these in the following PR